### ISSUE TITLE
Update Unique Humanity Scoring Model with CyberConnect Stamp Integration

### ISSUE THREAD

**Issue by Jeremy-Gitcoin** [gitcoinco/passport#1449]
### User Story:

As a user of CyberConnect / Link3 (https://cyberconnect.me/mint), 
I want to verify my Cyberprofile
So that I can prove I am a unique human and earn more stamps.

### Acceptance Criteria
GIVEN I have minted a Cyberprofile at CyberConnect (https://cyberconnect.me/mint),
WHEN I verify my CyberConnect stamp,
THEN I should earn the CyberConnect credentials with a weight of 1.21 in the Unique Humanity Scoring model.

The 'tweets on distinct days' Twitter credentials should be removed from the scoring model.

**Verifications / Credentials:**

Name: CyberConnect
Description: Connect your wallet to verify your CyberProfile Handle.

Three credentials:

- Premium CyberProfile Handle
- Paid CyberProfile Handle
- Organization Membership

#### Tech Details:
CyberConnect team has created PR 1353 - https://github.com/gitcoinco/passport/pull/1353

#### Open Questions:

- [ ] Cyberconnect will be running analysis on all 3 of the credentials to determine bot/sybil rate based on internal data
- [ ] Please contact Jeremy to be added to the CyberConnect Telegram group to work with Peiwen, the developer on Cyberconnect.

#### Notes/Assumptions:
The scoring model for the Unique Humanity Score will be updated to assign a weight of 1.21 to each of the CyberConnect credentials. The 'tweets on distinct days' Twitter credentials will be removed from the scoring model.

**Comment by lucianHymer** [comment]
We should have them use the context to cache the result of the contract call

**Comment by schultztimothy** [comment]
Would you all like to have separate inputs for each VC?

![Image](https://github.com/gitcoinco/passport/assets/6887938/9ebef382-e2e5-463e-85cc-9589896270a4)

**Comment by schultztimothy** [comment]
I created a new PR with these changes here https://github.com/gitcoinco/passport/pull/1557 I optimized it a bit to use context for duplicate requests and changed the tests so that they no longer make external requests.

I was able to impersonate the address that were provided in the tests and was able ensure the criteria is met. I don't have any funds on binance so was not able to test the criteria manually.

@erichfi and @Jeremy-Gitcoin are you by chance familiar with the platform? I wrote one comment in https://github.com/gitcoinco/passport/pull/1353 I'm a little confused by the criteria. I guess if you have a paid or premium account you get a profile with a certain length? If you all are cool with it I am!
